sidm/tools/scaleout.py

```diff
@@ -7,6 +7,7 @@ def make_dask_client(address):
     """Create dask client that includes dependency installer"""
     dependencies = [
         "git+https://github.com/btcardwell/SIDM.git",
+        #"git+https://github.com/yfv2ev/SIDM.git#bgMonteCarlo"
```

Comment:
Hi @yfv2ev, would you mind removing this line? It totally makes sense what you're up to, but I don't think we need to add it to the main branch

Reply:
Fixed